addons/l10n_eg_edi_eta/static/src/js/sign_invoice.js

```diff
@@ -0,0 +1,86 @@
+odoo.define('l10n_eg_edi_eta.action_post_sign_invoice', function (require) {
+    const core = require('web.core');
+    const ajax = require('web.ajax');
+    const Dialog = require('web.Dialog');
+    var rpc = require('web.rpc');
+    var _t = core._t;
+
+    function get_drive_error(value) {
+        switch(value) {
+           case 'missing_dll': return _t("Missing DLL - Make sure eps2003csp11.dll is correctly installed. You can download it here: https://www.egypttrust.com/en/downloads/other-drivers");
```

Comment:
This could happen also on Linux/Mac, so the instructions should cover other platforms as well, no? Also it might be better to point to a page in our documentation instead - it will be easier to maintain when the URLs go 404.